docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/focalmeter/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Kantar FocalMeter"
+date: "2023-01-16"
```

Comment:
It’s a well kept secret that this field does not actually do anything in Docusaurus, as it gets the date from git. I recommend not adding `date` to new pages to avoid confusion. The only reason I didn’t remove it for all pages is to prevent bumping the date everywhere (due to git changes).